robottelo/ui/repository.py

```diff
@@ -40,6 +40,9 @@ def create(self, name, product=None, gpg_key=None, http=False, url=None,
                 Select(type_ele).select_by_visible_text(gpg_key)
             if url:
                 self.text_field_update(locators["repo.url"], url)
+            if upstream_repo_name:
```

Comment:
It's probably a bit nicer to write `if upstream_repo_name is not None:`. However, I see that this idiom of `if method_argument:` is used quite a bit, so I'm not going to block on account of this.